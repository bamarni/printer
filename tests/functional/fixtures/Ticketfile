INIT

CHARSET PC850

ALIGN CENTER

PRINT Être ou ne pas être

ALIGN LEFT

FONT B

PRINTRAW
This        is
multiline
>>>

CUT
