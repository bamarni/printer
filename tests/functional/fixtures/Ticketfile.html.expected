<div style="text-align:center;color:black">Être ou ne pas être</div><div style="text-align:left;color:black">This &nbsp; &nbsp; &nbsp; &nbsp;is<br />multiline<br /></div><hr />